vcpkg/get_started/get-started.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Install and use packages with vcpkg
+description: Tutorial guides the user through the process of installing and using packages with vcpkg.
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: operating-systems
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages with vcpkg
+
+In this tutorial you'll create a C++ "Hello World" CMake project using vcpkg and the fmt library. You'll manage dependencies and build and run a simple application.
+
+## Prerequisites
+
+- A terminal to work in.
+- `git`, `CMake`, and a C++ compiler installed on your system
+- vcpkg installed on your system
+
+## 1 - Set up the project
+
+1. Create the project directory
+
+    This tutorial assumes that your project will be located in a directory named `helloworld`. The `helloworld` directory should be a sibling (or peer) of your vcpkg directory, meaning both should reside under the same parent directory. If you intend to place your project somewhere else, replace any references to that path with your preferred path.
+    
+    To create the directory and navigate into it, run:
+
+    ```console
+    mkdir helloworld && cd helloworld
+    ```
+
+2. Create the manifest
+
+    In this step, you'll create a file named `vcpkg.json` in your project's directory. 
+    
+    From within the `helloworld` directory, run:
+
+    ```console
+    ../vcpkg/vcpkg new --application
+    ```
+
+    The `vcpkg new` command adds a `vcpkg.json` file and a `vcpkg-configuration.json` file to your projects directory. The `vcpkg.json` file should look like this:
+
+    ```json
+    {}
+    ```
+
+    This is your manifest file. vcpkg reads the manifest file to learn what dependencies to install. The manifest integrates vcpkg into your C++ project by providing a list of dependencies and supported features that are part of your project. Initially, the file contains an empty JSON object; additional attributes like dependencies and supported features can be added as your project grows.
+
+    The `vcpkg-configuration.json` file serves a different purpose. This file allows full control over your dependencies' sources and it's out of scope for this tutorial.
```

Comment:
The default `vcpkg-configuration.json` will add baseline constraints to the manifest, so we either make a very basic explanation of what that means or we tell the user to delete the file.